As part of its installation, RDKit downloads a font package from https://fonts.google.com
and checks its md5sum. It appears the font package was modified and therefore a new md5sum
must be used. The authors of RDKit have updated the md5sum in the source code of RDKit but
no backward fix was made for already existing releases. This patch updates the md5 checksum
to the its new value.
Author: Robin Engler, SIB Swiss Institute for Bioinformatics
--- rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt.orig	2022-01-17 13:56:43.780726092 +0100
+++ rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt	2022-01-17 13:57:02.668646178 +0100
@@ -12,7 +12,7 @@
     set(needDownload "FALSE")
   endif()
   if(needDownload)
-    set(MD5Sum "23ed3f833c1ae0adb141a26b4a30d73e")
+    set(MD5Sum "850b0df852f1cda4970887b540f8f333")
     downloadAndCheckMD5("https://fonts.google.com/download?family=Comic%20Neue"
           "${CMAKE_CURRENT_SOURCE_DIR}/Comic_Neue.zip"
           ${MD5Sum})
